list_updatable_packages

```diff
@@ -46,7 +49,10 @@ class Spec:
             self._globals = {}
             for line in self.lines:
                 if line.startswith('%global'):
-                    definition, name, value = line.split(None, 3)
+                    split_line = line.split()
+                    definition = split_line[0]
+                    name = split_line[1]
+                    value = split_line[2]
```

Comment:
What's the reason to modify this?

Reply:
This logic assumed that all globals followed the exact pattern of only having three arguments, that is not true and globals can have multiple arguments.

Reply:
That's why it was limited to splitting 3 times. It would only fail if there were fewer than 3, but that's invalid since `%global` is one, the name of the macro two and the value is 3. Am I missing something?

Reply:
The reason I hammer on this is that you are introducing a regression. Going through this I actually see I already had a bug though:
```
In [1]: '%global %mymacro "A macro with spaces"'.split(None, 3)
Out[1]: ['%global', '%mymacro', '"A', 'macro with spaces"']

In [2]: '%global %mymacro "A macro with spaces"'.split(None, 2)
Out[2]: ['%global', '%mymacro', '"A macro with spaces"']

In [3]: '%global %mymacro "A macro with spaces"'.split(None)
Out[3]: ['%global', '%mymacro', '"A', 'macro', 'with', 'spaces"']
```

I think this is because in Ruby you pass the desired number of results and in Python the number of times it splits

Reply:
Resolved that bug in https://github.com/theforeman/foreman-packaging/pull/8069